bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,5 +6,6 @@
 	<type>binding</type>
 	<name>Meteo Alerte Binding</name>
 	<description>Retrieves Meteo Alert levels in France</description>
+	<connection>cloud</connection>
```

Comment:
Done

Reply:
> You could consider also adding countries if this binding only applies to France - see https://next.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions. This can be done in a follow-up PR if relevant since we have a pending clarification: #14778

The clarification is still pending, but I'll merge this PR now and we can come back to this later.